tasks/main.yml

```diff
@@ -1,2 +1,184 @@
 ---
 
+- name: Install postfix
+  package:
+    name:
+      - postfix
+      - postfix-ldap
+      - postfix-pgsql
+    state: present
+
+- name: Setup LDAP authentication
+  block:
+    - name: Install saslauthd
+      package:
+        name:
+          - libsasl2-modules
+          - sasl2-bin
+        state: present
+
+    - name: Place LDAP connection config file
+      template:
+        src: saslauthd.conf.j2
+        dest: /etc/saslauthd.conf
+
+    - name: Place SMTP SASL config file
+      copy:
+        src: sasl-smtpd.conf
+        dest: /etc/postfix/sasl/smtpd.conf
+
+    - name: Place saslauthd-postfix config file
+      template:
+        src: saslauthd-postfix.j2
+        dest: /etc/default/saslauthd-postfix
+
+    - name: Create socket directory for saslauthd
+      shell: dpkg-statoverride --add root sasl 755 /var/spool/postfix/var/run/saslauthd
```

Comment:
Is `755`, necesarry here?
The [debian docs](https://wiki.debian.org/PostfixAndSASL) make it `710`.

Reply:
These instructions are for Debian 7. I tried it with 710 and postfix didn't authenticate users with a permission denied error.
In theory 750 should be enough, but this also did not work. I hadn't had the time to investigate further at the time, so I set it to 755.